python/kserve/kserve/storage/storage.py

```diff
@@ -170,6 +170,23 @@ def _download_s3(uri, temp_dir: str):
         if verify_ssl:
             verify_ssl = verify_ssl != "0"
             kwargs.update({"verify": verify_ssl})
+        # If verify_ssl is true, then check there is custom ca bundle cert
+        if verify_ssl:
+            global_ca_bundle_secret = os.getenv("CA_BUNDLE_SECRET_NAME")
+            if global_ca_bundle_secret:
+                isvc_aws_ca_bundle_path = os.getenv("AWS_CA_BUNDLE")
+                if isvc_aws_ca_bundle_path:
+                    ca_bundle_full_path = isvc_aws_ca_bundle_path
+                else:
+                    global_ca_bundle_volume_mount_path = os.getenv("CA_BUNDLE_VOLUME_MOUNT_POINT")
+                    ca_bundle_full_path = global_ca_bundle_volume_mount_path + "/cabundle.crt"
```

Comment:
Should we support more than one bundle?

Reply:
No. I don't think there is a need to add more than 1 bundle.